examples/azure-rules.yaml

```diff
@@ -656,4 +656,17 @@ groups:
           Please, check the naming convention in case the resource configuration is done correctly.
            
           More information at https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability#create-sap-hana-cluster-resources
-        scored: true
\ No newline at end of file
+        scored: true
+      - id: 1.5.7
+        description: "Check /hana/share/myHooks/global.ini"
+        audit: 'cat /hana/share/myHooks/global.ini'
```

Comment:
Actually, this is not the location where global.ini resides, but where the python hook will be, so this check will always fail.

Reply:
Thanks, fixed. It searches the global.ini in /hana/shared/<SID>/global/hdb/custom/config/ now.